dace/transformation/transformation.py

```diff
@@ -224,7 +224,7 @@ def apply_pattern(self, append: bool = True, annotate: bool = True) -> Union[Any
         """
         if append:
             self._sdfg.append_transformation(self)
-        tsdfg: SDFG = self._sdfg.sdfg_list[self.sdfg_id]
+        tsdfg: SDFG = self._sdfg.cfg_list[self.cfg_id]
```

Comment:
How do we know it’s an SDFG? seems like it’s the parent cf block

Reply:
My bad, we do not. That is by design, too, but I am mixing PRs here a bit. In a different PR the interface for the transformation does not expect an SDFG anymore but a CFG (same for propagation etc.). However, I have now inserted a compatibility fix in this PR.